wear/src/main/java/io/homeassistant/companion/android/util/CommonFunctions.kt

```diff
@@ -56,15 +90,31 @@ fun getIcon(entity: Entity<Map<String, Any>>?, domain: String, context: Context)
                 CommunityMaterial.Icon2.cmd_light_switch
             }
             "input_button" -> CommunityMaterial.Icon2.cmd_gesture_tap_button
+            "input_datetime" -> CommunityMaterial.Icon.cmd_calendar_clock
+            "input_select" -> CommunityMaterial.Icon2.cmd_format_list_bulleted
+            "input_text" -> CommunityMaterial.Icon2.cmd_form_textbox
             "light" -> CommunityMaterial.Icon2.cmd_lightbulb
             "lock" -> when (compareState) {
                 "unlocked" -> CommunityMaterial.Icon2.cmd_lock_open
                 "jammed" -> CommunityMaterial.Icon2.cmd_lock_alert
                 "locking", "unlocking" -> CommunityMaterial.Icon2.cmd_lock_clock
                 else -> CommunityMaterial.Icon2.cmd_lock
             }
-            "script" -> CommunityMaterial.Icon3.cmd_script_text_outline // Different from frontend: outline version
+            "mailbox" -> CommunityMaterial.Icon3.cmd_mailbox
+            "notify" -> CommunityMaterial.Icon.cmd_comment_alert
+            "number" -> CommunityMaterial.Icon3.cmd_ray_vertex
+            "persistent_notification" -> CommunityMaterial.Icon.cmd_bell
+            "person" -> CommunityMaterial.Icon.cmd_account
+            "plant" -> CommunityMaterial.Icon2.cmd_flower
+            "proximity" -> CommunityMaterial.Icon.cmd_apple_safari
+            "remote" -> CommunityMaterial.Icon3.cmd_remote
             "scene" -> CommunityMaterial.Icon3.cmd_palette_outline // Different from frontend: outline version
+            "script" -> CommunityMaterial.Icon3.cmd_script_text_outline // Different from frontend: outline version
+            "select" -> CommunityMaterial.Icon2.cmd_format_list_bulleted
+            "sensor" -> CommunityMaterial.Icon.cmd_eye
+            "siren" -> CommunityMaterial.Icon.cmd_bullhorn
+            "simple_alarm" -> CommunityMaterial.Icon.cmd_bell
+            "sun" -> CommunityMaterial.Icon3.cmd_white_balance_sunny

```

Comment:
This one changes [if it is night](https://github.com/home-assistant/frontend/blob/ffc4ca5b568ad56754967462e9dca01bae2bd70d/src/common/entity/domain_icon.ts#L151-L154) in the frontend